### DISCUSSION TITLE
Add icon near row number

### DISCUSSION THREAD

**Discussion by andreymarchenko** [microsoft/monaco-editor#3734] [Q&A]
Hello
Is it possible to customize error highliting? 
E.g. add icon with error near row number (Can i do it by editor.deltaDecorations?)

![image](https://user-images.githubusercontent.com/17382387/62870917-22641480-bd23-11e9-827b-aaf235e74986.png)

**Comment by sundaycrafts** [comment]
This issue is already done?

see https://microsoft.github.io/monaco-editor/playground.html#interacting-with-the-editor-rendering-glyphs-in-the-margin

I haven't tried yet but I think we can add any images by CSS background-image.

![image](https://user-images.githubusercontent.com/4732821/107144068-f824fe80-697b-11eb-8d0f-d6f78ee0fe67.png)
